lib/keybase-exec.js

```diff
@@ -5,12 +5,13 @@ import readline from 'readline'
 type ExecOptions = {|
   stdinBuffer?: Buffer | string,
   onStdOut?: (line: string) => void,
+  parseOutputAsJSON?: boolean,
```

Comment:
If we're parsing JSON a lot (which we will be for chat), we could make this a negative. I suppose we could also leave the JSON parsing to be implicit, but that almost feels too magical to me.

Reply:
I see what you mean here.
Quick thing - perhaps change this flag to just `json?: boolean` to reduce the function signature?

Since the chat API methods are funneled through `runApiCommand` we can always pass `json: true` right? That basically sets it for every call to `keybase chat api`. Also no one using this library will be calling `keybaseExec` directly so this shouldn't be too much of an issue.

I think it's safer to leave the flag as a positive because it would be more confusing to set the default to `true`, have someone forget to turn it off, parse the non JSON output as JSON, get a `JSON.parse()` error` and then be confused.